packages/components/src/RBACProvider/RBACProvider.tsx

```diff
@@ -10,7 +9,7 @@ import {
   initialPermissions,
 } from '@redhat-cloud-services/frontend-components-utilities/RBAC';
 
-const hasAccessWithUserPermissions = (userPermissions: (Access | string)[]) => {
+const hasAccessWithUserPermissions = (userPermissions: string[]) => {
```

Comment:
Bot of the `hasAllPermissions` and `doesHavePermissions` accept the `Access[]` type as well as `string[]` for the first argument. Any reason why we don't want it here?

```TS
export function hasAllPermissions(userPermissions: (Access | string)[], permissionList: string[]): boolean {

export function doesHavePermissions(userPermissions: (Access | string)[], permissionList: string[]): boolean {

```

Reply:
Thanks for the note! I'll update that as well. I'm just not as confident with  my TS to see these things right away, yet. :)

Reply:
If you are re-using anying that is already written in TS, the best bet is just to re-use the same TS definition for arguments.